API.md

```diff
@@ -107,11 +107,11 @@ __Parameters__
 |:--- |:--- |:--- |
 | ``bucketName``  | _string_ | Name of the bucket.  |
 | ``region``  | _string_| Optional parameter. Defaults to us-east-1 for AWS requests  |
-
+| ``cancellationToken``| System.Threading.CancellationToken | Optional parameter. Defaults to default(CancellationToken) |
```

Comment:
In other SDK API doc, we use `_datatype_` to mark-up types. So, `System.Threading.CancellationToken` will be `_System.Threading.CancellationToken_`.